gulp/makeCss.js

```diff
@@ -62,3 +62,7 @@ gulp.task("copyCssDeps", function() {
     )
     .pipe(gulp.dest("./src/scss/cssIncludes"));
 });
+gulp.task('copyCssImages', function() {
+   gulp.src('./node_modules/leaflet/dist/images/*.png')
+   .pipe(gulp.dest(paths.bundleDir+'/images'));
+});
```

Comment:
But if I include YASGUI (with YASR as dep) in another library, then I'd need to do more than just pull in the js and css as dep. Now I'd also need to make sure I've downloaded the corresponding img assets, and host them correctly. That is an unnecessary burden to put on other developers, as it's something we can avoid with little cost (the base encoded image is only ~300 bytes larger than the actual png)